### PR TITLE
Portfolio setup and mobile first-Feature 1

### PR DESCRIPTION
**Pull request by MANYIKAM**
My Portfolio setup and mobile first.
The following were done during this session
1. Set up of the Repo
2. Readme file has been created and updated inline with the project
3. Html file has been developed as per the requirement of having the first two sections
4. Css style sheet has been created and updated accordingly
5. Linters errors has been  done

### CODE REVIEW

**Review by GiuseppeTG — CHANGES_REQUESTED** [review]
Hi @MANYIKAM  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ 

You have submitted a project, that has not properly set up linters. As you can see, in this PR linters checkers are not running. 
You need to have this folder structure for your linters: 
- .github > workflows > linters.yml

Please check the [Microverse Linters](https://github.com/microverseinc/linters-config) repo for more information 🤓 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio as a webdesigner</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/69bb18a36d.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="flex">
+        <h1 class="log">Manyikam</h1>
```
Comment (frank1738) [inline index.html:13]: - [ ] Kindly wrap your logo in an `<a>` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <h2 class="social-media">LET’S CONNECT</h2>
+     <ol class="social-meadia-option">
+        <li><i class="fa-brands fa-twitter"></i>
+            <li><i class="fa-brands fa-linkedin"></i></li>
+            <li><i class="fa-solid fa-m"></i></li>
+            <li><i class="fa-brands fa-github"></i></li>
+            <li><i class="fa-regular fa-hand-peace"></i></li>
+        
+    </ol>
+</section>
```
Comment (frank1738) [inline index.html:31]: - [ ] Kindly Provide your design with background images to match the Figma design
### Your Desin

image

### Figma Design

image

- [ ] Kindly space your social icons to ensure proper spacing of elements to match the Figma design

image

### Check Figma

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @MANYIKAM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅Good PR summary👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <nav>
+    <i class="fa-solid fa-bars"></i>
+        </nav>
+</header>
+<section class="details">
+    <h1 class="owner">I’m Manyikam,
+        Glad to see you at my website Portfolio!</h1>
+    <p class="descprition">I’m a software developer! I can help you build a product , feature or website. You can Look through some of my work and experience on this portfolio! If you like what you see and have a project you need to be coded, don’t hestiate to contact me.</p>
+    <h2><a class="social-media" href="#">LET’S CONNECT</a></h2>
+     <div class="flexbox">
```
Comment (jasonfelice) [inline index.html:23]: - [ ] Kindly use `ul` to group social links.
- [ ] The social links don't look like the Figma design. Please fix that.
image

File: Style.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  line-height: 52px;
+  color: #172b4d;
+}
+
+body {
+  background-image: url('img/bground-mobile1.png');
+  border-radius: 0 0 0 50px;
+  background-color: #fff;
+  padding:19px;
+}
```
Comment (jasonfelice) [inline Style.css:76]: - [ ] The design doesn't look like the given Figma template. Kindly remove the background from the `body`, and use it on appropriate section(s). 😄 
image

## Figma Design:
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        <h1><a class="log" href="#">Manyikam</a></h1>
+        <nav>
+    <i class="fa-solid fa-bars"></i>
+        </nav>
+</header>
+<section class="details">
+    <h1 class="owner">I’m Manyikam,
+        Glad to see you at my website Portfolio!</h1>
+    <p class="descprition">I’m a software developer! I can help you build a product , feature or website. You can Look through some of my work and experience on this portfolio! If you like what you see and have a project you need to be coded, don’t hestiate to contact me.</p>
+    <h2><a class="social-media" href="#">LET’S CONNECT</a></h2>
```
Comment (jasonfelice) [inline index.html:22]: - [ ] Let's use proper style and color for this link. Please refer to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @MANYIKAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
- [ ] Please make sure the linter checks are running and there are no errors.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by isaacpitwa — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @MANYIKAM   👋🏻 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MANYIKAM** [comment]
Portfolio Setup and mobile first -feature1 merging with the main 

- Html file has been updated with all required changes
- Css file has been updated accordingly with few changes
- Linters has been set up as per the requirement

**Pull request merged**